app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -62,18 +61,28 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         forgotPasswordProgressDialog.setView(forgotPasswordProgressDialog.layoutInflater.inflate(R.layout.progress, null))
 
         addListeners()
-
         cancelRequestPassword()
         requestPassword()
-
+        remember()
         loginPresenter = LoginPresenter(this)
         loginPresenter.onAttach(this)
+
         val bundle = intent.extras
         val string = bundle?.getString("email")
         if (string != null)
             email.editText?.setText(string)
     }
 
+    private fun remember() {
+        val saveLoginCheckBox = findViewById<CheckBox>(R.id.remember)
+
+        if (getSharedPreferences("loginPrefs", MODE_PRIVATE).getBoolean("saveLogin", false) == true) {
+            email.editText?.setText(getSharedPreferences("loginPrefs", MODE_PRIVATE).getString("username", ""))
+            password.editText?.setText(getSharedPreferences("loginPrefs", MODE_PRIVATE).getString("password", ""))
+            saveLoginCheckBox.setChecked(true)
```

Comment:
Won't save the email and password in shared preferences. Highly insecure

Reply:
Can you please help me out @iamareebjamal  for a better option.